.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.1
```

Comment:
it's supposed to run on any ruby version, so should not need pinning

Reply:
Makes sense. Removing file. (This was added by the original author, @chamnap, not me.)